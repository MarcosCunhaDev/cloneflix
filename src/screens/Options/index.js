import React from "react";
import { View, Text } from "react-native";
import ContainerDefault from "../../components/Container";

function OptionsScreen() {
  return (
    <ContainerDefault>
      <Text>Options Screen</Text>
    </ContainerDefault>
  );
}

export default OptionsScreen;
