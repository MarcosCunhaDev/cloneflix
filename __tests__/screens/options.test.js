import React from "react";
import renderer from "react-test-renderer";

import Options from "../../src/screens/Options";

describe("<Options />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<Options />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

it("renders correctly", () => {
  const tree = renderer.create(<Options />).toJSON();
  expect(tree).toMatchSnapshot();
});
