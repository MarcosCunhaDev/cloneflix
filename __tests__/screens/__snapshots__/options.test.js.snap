// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#121921",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-start",
        "paddingLeft": 16,
        "paddingTop": 40,
      },
    ]
  }
>
  <Text>
    Options Screen
  </Text>
</RCTSafeAreaView>
`;
